roles/collect_modified_images/tasks/parse_image_names.yml

```diff
@@ -0,0 +1,9 @@
+- name: "split item"
+  set_fact:
+    timestamp: "{{ item.split(',')[0] }}"
+    image: "{{ item.split(',')[1][15:-1] }}"
```

Comment:
Such parsing is very dangerous. Input can change due to various reasons. And we might get unexpected results
due to parsing without any validation.

Reply:
We're collecting the results as a json file [here](https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/4fba992ef95962781bf826da455d8275b1f65500/roles/collect_modified_images/tasks/main.yml#L8) and simply outputting the fields as csv:

```
timestamp1,message1
timestamp2,message2
...
```

The format of the timestamp or message could change, but I don't see a way around that if we're going to parse the events.json.

Reply:
that is explanation for the 1st index in the split.
but does not explain why do we need `[15:-1]`.

Please put examples as comments into the code.
So sb else will be able to change/fix it if we need. Adding checks for expected format will be even better.